### PR TITLE
Add `-` to powershell identifiers

### PR DESCRIPTION
**Pull request by efritz**

### CODE REVIEW

**Review by Strum355 — APPROVED** [review]
Is this really the entire fix lmao

File: shared/language-specs/languages.ts [hunk 1]
```diff
@@ -202,7 +202,7 @@ const powershellSpec: LanguageSpec = {
     languageID: 'powershell',
     stylized: 'PowerShell',
     fileExts: ['ps1', 'psd1', 'psm1'],
-    identCharPattern: createIdentCharPattern('?'),
+    identCharPattern: createIdentCharPattern('-?'),
```
Comment (efritz) [inline shared/language-specs/languages.ts:205]:
```suggestion
    identCharPattern: createIdentCharPattern('?-'),
```

**Comment by efritz** [comment]
@Strum355 **No**, I just noticed that this is wrong too and it's a common identifier in functions.

**Comment by keegancsmith** [comment]
Don't know powershell at all, so don't know if this is correct but assume it is :)

**Pull request merged**